dev_guide/console.adoc

```diff
@@ -69,34 +69,3 @@ image:console_filter_step3.png["Console Filter Step 3"]
 
 image:console_filter_active.png["Console Filter Active"]
 ====
-
-
-== Creating from Template
```

Comment:
if we're getting rid of the console section entirely I guess that's ok (sorry I didn't catch up on all the iteration that occurred while I was out) then...but I was assuming and think it would be better if we still have a section dedicated to the console that would cover at a high level what you can do from the console.    As a user I don't want to have to read through specific sections to find out what i can do in the console.  **How** I do those things in the console belong in the specific sections.